src/lib/category.ml

```diff
@@ -0,0 +1,68 @@
+
+(* This file is free software, part of FTW. See file "LICENSE" for more information *)
+
+(* Categories:
+
+   These are the categories that a competition can have:
+
+   - either a competitive division (i.e. one which gives points)
+   - or a non-competitive one (no points given or required), which has a
+     few different distinctions, most notably related to the start of
+     the point system
+*)
```

Comment:
should we add this comment about categories into the concept.md file ?

Reply:
I'll do that indeed